### PR TITLE
feat(Must): create extension

### PR DESCRIPTION
**Pull request by eggsy**
## Description

Created an extension for [Must](https://mustapp.com/), a great app to keep track of the movies/series you watched, and the ones you will watch in the future.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/13917975/149185404-b20a463d-d0e4-48e4-b8c8-abe19795c684.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by CanCodes — APPROVED** [review]
LGTM! Looking forward to using this extension!

**Comment by Merloss** [comment]
hmm, lgtm

File: extensions/must/package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "keywords": [
+    "must",
+    "movie",
+    "series",
+    "tracking"
+  ],
+  "commands": [
+    {
+      "name": "index",
+      "title": "Must",
```
Comment (pernielsentikaer) [inline extensions/must/package.json:18]:
```suggestion
      "title": "Watch list",
      "subtitle": "Must",
```

File: extensions/must/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      "description": "Your want to watch list on Must.",
+      "subtitle": "Your want to watch list on Must.",
```
Comment (pernielsentikaer) [inline extensions/must/package.json:20]:
```suggestion
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @eggsy 👋 

Awesome addition, looking forward to use it 🔥 

If I insert a username that don’t exists, it loads forever - here it should show a Failure Toast with a reason and the loader should be stopped with empty array.

Can you look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by eggsy** [comment]
Hey @pernielsentikaer, sorry I had to commit via GitHub web because I have other files (extension) that are waiting to be pushed and I didn't want to include them in this PR 😅. It works as you requested now. You can take a look at it again!

**Review by pernielsentikaer — APPROVED** [review]
Hi @eggsy 👋 

MUCH better 🔥 I'm going to accept it and then will my colleague do a final review before it's finally merged – well done! 💎

**Pull request merged**